src/applications/edu-benefits/components/EducationWizard.jsx

```diff
@@ -447,7 +447,7 @@ export class EducationWizard extends React.Component {
             {newBenefit === 'no' &&
               (transferredEduBenefits === 'transferred' ||
                 transferredEduBenefits === 'own') &&
-              this.getButton(form1995)}
+              this.getButton('1995')}
```

Comment:
Undo this change, this is a place where the feature flag can continued to be used and is unrelated to this story

Reply:
This was a part of Dustin's 10203 change, should it be left off for now?

Reply:
Found the conversation about including this change, this is ok to leave in